apis/Google.Cloud.Spanner.V1/Google.Cloud.Spanner.V1/SessionPool.cs

```diff
@@ -38,7 +38,7 @@ namespace Google.Cloud.Spanner.V1
     /// and it will incur a penalty on other Spanner processes because it will be an hour before the server frees the session if 
     /// it's not properly deleted.
     /// </summary>
-    internal static class SessionPool
+    public static class SessionPool
```

Comment:
in an ideal scenario, you'd be able to turn on session pooling and use the same apis you would normally -- but the sessions would be pooled.  the problem with doing this is with modifying (or hooking into) generated code.

So I'm not sure I'm completely happy with the current api and how someone would use the pool directly at the V1 level.  But at the same time, there probably are going to be very few folks who need to...

It would just be nice to give a little more thought on how folks would use this functionality.